lib/ts/recipe/emailpassword/types.ts

```diff
@@ -105,35 +121,35 @@ export type RecipeInterface = {
         userContext: any;
     }): Promise<{ status: "OK"; user: User } | { status: "WRONG_CREDENTIALS_ERROR" }>;
```

Comment:
sign in, sign up  for all recipes should also return the recipeUserId along with the user -> { status: "OK"; user: User, recipeUserId: RecipeUserId }